client/apps/edit/components/content/section_list/index.jsx

```diff
@@ -86,11 +97,24 @@ export class SectionList extends Component {
           sections={sections}
           index={-1}
           key={1}
-          isEditing={editingIndex !== null}
+          isEditing={activeSection !== null}
           firstSection
           isDraggable={false}
         />
       </div>
     )
   }
 }
+
+const mapStateToProps = (state) => ({
+  ...state
+})
+
+const mapDispatchToProps = (dispatch) => ({
+  actions: bindActionCreators(Actions, dispatch)
+})
```

Comment:
Ah, you don't necessarily need `bindActionCreators` here as you can assign specific actions and they're bound for you. For example, there are to possible forms:

```javascript
import { changeSection } from 'my/actions'

const mapDispatchToProps = {
  changeSectionAction: changeSection
}
```
(Its nice to add "Action" to the end of the key so that it jumps out in props code, though not required.)

Can also do (similar to what you have): 

```javascript
const mapDispatchToProps = (dispatch) => {
  const dontDispatchRightAway = debounce(() => dispatch(myAction), 1000)

  return {
    someOtherAction: dontDispatchRightAway
  }
}
```
The second form is good if you need to perform intermediate actions before things are dispatched, or mess with the action pipeline in some way.

Reply:
Excellent, thanks!